Ryujinx.Ui.Common/App/ApplicationData.cs

```diff
@@ -19,5 +30,128 @@ public class ApplicationData
         public double FileSizeBytes { get; set; }
         public string Path          { get; set; }
         public BlitStruct<ApplicationControlProperty> ControlHolder { get; set; }
+        public static string GetApplicationBuildId(VirtualFileSystem virtualFileSystem, string titleFilePath)
```

Comment:
```suggestion

        public static string GetApplicationBuildId(VirtualFileSystem virtualFileSystem, string titleFilePath)
```
Since this is a method and not a property, add a empty line to make these easier to distinguish.